cirq-core/cirq/transformers/analytical_decompositions/__init__.py

```diff
@@ -66,6 +66,7 @@
 from cirq.transformers.analytical_decompositions.two_qubit_state_preparation import (
     prepare_two_qubit_state_using_cz,
     prepare_two_qubit_state_using_sqrt_iswap,
+    prepare_two_qubit_state_using_iswap,
```

Comment:
You also need to add it to the top-level `cirq/__init__.py` so that `cirq.prepare_two_qubit_state_using_iswap` is available. This is causing your tests to fail.